### PR TITLE
tidb replica read support global set.

### PR DESCRIPTION
**Pull request by tier-cap**
Signed-off-by: tier-cap <zhengxiaojin@pingcap.com>

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

tidb replica read support global set.
### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v5.4 (TiDB 5.4 versions)
- [x] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/8422
- Other reference link(s): 
https://github.com/pingcap/docs/pull/6451/files
https://github.com/pingcap/tidb/pull/27622

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by xhebox — APPROVED** [review]

File: follower-read.md [hunk 1]
```diff
@@ -23,14 +23,14 @@ To enable TiDB's Follower Read feature, modify the value of the `tidb_replica_re
 {{< copyable "sql" >}}
```
Comment (en-jin19) [inline follower-read.md:23]: @tier-cap please update line 21 from "To enable TiDB's Follower Read feature, modify the value of the `tidb_replica_read` session variable:" to "To enable TiDB's Follower Read feature, modify the value of the `tidb_replica_read` variable:"

**Review by en-jin19 — CHANGES_REQUESTED** [review]
line 21 is to be changed to remove "session"

**Review by NingLin-P — APPROVED** [review]

File: follower-read.md [hunk 2]
```diff
@@ -18,19 +18,19 @@ The Follower Read feature refers to using any follower replica of a Region to se
 
 ## Usage
 
-To enable TiDB's Follower Read feature, modify the value of the `tidb_replica_read` session variable:
+To enable TiDB's Follower Read feature, modify the value of the `tidb_replica_read` variable:
```
Comment (en-jin19) [inline follower-read.md:21]:
```suggestion
To enable TiDB's Follower Read feature, modify the value of the `tidb_replica_read` variable to `follower` or `leader-and-follower`:
```

File: follower-read.md [hunk 3]
```diff
@@ -18,19 +18,19 @@ The Follower Read feature refers to using any follower replica of a Region to se
...
+set [session | global] tidb_replica_read = '<target value>';
 ```
 
-Scope: SESSION
+Scope: SESSION | GLOBAL
 
 Default: leader
 
-This variable is used to set the data read mode expected by the current session.
+This variable is used to set the data read mode.
```
Comment (en-jin19) [inline follower-read.md:33]:
```suggestion
This variable is used to set the expected data read mode.
```

**Review by en-jin19 — APPROVED** [review]
LGTM

**Pull request merged**